Gallery/gallery-index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./gallery-style.css">
+    <link rel="stylesheet" href="../style.css">
+    <title>Gallery</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <h1>Pet Vogue</h1>
+            <li>
+                <ul id="main-nav"><a href="./Appointment/ContactMe.html"><button>APPOINTMENTS</button></a> </ul>
+                <ul id="main-nav"><a href="./about/aboutPage.html"><button>ABOUT</button></a></ul>
+                <ul id="main-nav"><a href="../Staff/staff-index.html"><button>STAFF</button></a></ul>
+                <ul id="main-nav"><a href="../index.html"><button>HOME</button></a></ul>
+            </li>
+        </nav>
+        <section>
+            <main>
+                <ul>
+                    <li><img src="../images/gallery/gallery (1).jpg" alt=""></li>
```

Comment:
when naming files, fill the spaces with hyphens` - `or underscores` _`
`../images/gallery/gallery-1.jpg`